### PR TITLE
[SR-22] Added additional field for User model

### PR DESCRIPTION
**Pull request by AKShipulya**
## Note
* necessary fields added
* additionally were changed service and controller methods names, because in my opinion there is no needed word "user" in naming because service and controller names already contains that (**User**Controller, **User**Service)

### CODE REVIEW

File: src/main/java/com/spotride/spotride/user/controller/UserController.java [hunk 1]
```diff
@@ -36,8 +35,8 @@ public final class UserController {
     UserService userService;
 
     @GetMapping
-    public List<UserResponseDto> getAllUsers() {
```
Comment (N0TARGET) [inline src/main/java/com/spotride/spotride/user/controller/UserController.java:40]: Why do we need this renaming?

File: src/test/java/com/spotride/spotride/user/service/UserServiceTest.java [hunk 2]
```diff
@@ -158,18 +180,18 @@ void testUpdateUser() {
...
+        var result = userService.update(1L, userUpdateRequestDto);
 
         assertNotNull(result);
         assertEquals("john_updated", result.getUsername());
         verify(mockUserRepository, times(1)).save(user);
     }
 
     @Test
-    void testDeleteUser() {
+    void testDelete() {
```
Comment (N0TARGET) [inline src/test/java/com/spotride/spotride/user/service/UserServiceTest.java]: testDelete what? please return initial naming

File: src/main/resources/database/liquibase/changesets/user/changelog.xml [hunk 3]
```diff
@@ -27,6 +27,13 @@
             <column name="modified_at" type="TIMESTAMP">
                 <constraints nullable="true"/>
             </column>
+            <column name="birth_date" type="DATE"/>
```
Comment (N0TARGET) [inline src/main/resources/database/liquibase/changesets/user/changelog.xml:30]: This is incorrect modification. For each new change additional changeset should be created. Need to have a call.

File: src/test/java/com/spotride/spotride/user/service/UserServiceTest.java (hunk 2 above)
Comment (AKShipulya) [reply src/test/java/com/spotride/spotride/user/service/UserServiceTest.java]: fixed

File: src/main/resources/database/liquibase/changesets/user/changelog.xml (hunk 3 above)
Comment (AKShipulya) [reply src/main/resources/database/liquibase/changesets/user/changelog.xml:30]: discussed it, leave it at that

File: src/main/java/com/spotride/spotride/user/controller/UserController.java (hunk 1 above)
Comment (AKShipulya) [reply src/main/java/com/spotride/spotride/user/controller/UserController.java:40]: discussed it, leave it at that

**Review by N0TARGET — APPROVED** [review]

**Pull request merged**